prov/efa/Makefile.include

```diff
@@ -47,13 +47,15 @@ _efa_files = \
 	prov/efa/src/rxr/rxr_cq.c	\
 	prov/efa/src/rxr/rxr_ep.c	\
 	prov/efa/src/rxr/rxr_cntr.c	\
-	prov/efa/src/rxr/rxr_rma.c
+	prov/efa/src/rxr/rxr_rma.c	\
+	prov/efa/src/rxr/rxr_msg_ops.c
```

Comment:
Nit: Maybe just name it rxr_msg.c, in line with the other sources?

Reply:
I have follow up commit that rely on this name. Is it OK to use the name for now and change it to rxr_msg.c after they are merged?

Reply:
Per offline discussion with @rajachan, change name later would cause some issues, so I uploaded a new revision with the file name changed to rxr_msg.c, commit message and Makefile.include have also been updated.